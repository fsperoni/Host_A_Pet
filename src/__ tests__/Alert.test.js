import React from 'react';
import { render } from "@testing-library/react";
import Alert from '../components/Alert';

const messages = ["Hello!"]
// smoke test
it('renders without crashing', function() {
  render(<Alert type="danger" messages={messages} />);
});

//snapshot test
it("matches snapshot", function() {
  const {asFragment} = render(<Alert type="danger" messages={messages}/>);
  expect(asFragment()).toMatchSnapshot();
});
