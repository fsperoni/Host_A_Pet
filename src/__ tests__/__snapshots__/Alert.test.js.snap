// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="alert mt-3 alert-danger"
    role="alert"
  >
    <p
      class="mb-0 small"
    >
      Hello!
    </p>
  </div>
</DocumentFragment>
`;
